dexlib2/src/main/java/org/jf/dexlib2/dexbacked/MappedDexBuffer.java

```diff
@@ -0,0 +1,104 @@
+package org.jf.dexlib2.dexbacked;
```

Comment:
As per my last comments, I don't think class needs to be submitted to the smali repo

Reply:
Done. Thanks